types/tests/server-test.ts

```diff
@@ -36,7 +37,14 @@ export default function config(this: Server): void {
   this.get<[number, string]>("/foo", () => ["foo", 0]); // $ExpectError
 
   this.passthrough("/_coverage/upload"); // $ExpectType void
+  this.passthrough("/_coverage/upload_a", "/_coverage/upload_b"); // $ExpectType void
+  this.passthrough(["/_coverage/upload"]); // $ExpectError
   this.passthrough((request) => request.queryParams.skipMirage); // $ExpectType void
+  this.passthrough("/_coverage/upload", ["get"]); // $ExpectType void
+  // prettier-ignore
+  this.passthrough("/_coverage/upload", (request) => request.queryParams.skipMirage); // $ExpectType void
+  // prettier-ignore
```

Comment:
prettier-ignore is required otherwise we will get line break and dtslint will not understand end line annotation